### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
The README hasn't been touched in a while and we have done a bunch of stuff on the product. This change to the REAMDE reflects those product updates. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,27 @@
...
+Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to help developers go beyond Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
 
 ## Features
 
-Watermelon has 2 actions that you can use:
+Watermelon has 3 main features that you can use:
 
 - View Git Blame
 - View Pull Requests
+- Daily Summary
```
Comment (EstebanDalelR) [inline README.md:22]: This should only be merged when #350 gets released

Comment (EstebanDalelR) [reply README.md:22]: The current state of daily summary is *barely usable*

File: README.md [hunk 2]
```diff
@@ -11,16 +11,27 @@
...
 
-Simply click the Watermelon icon on the sidebar. After that you can click any of the 2 different buttons Watermelon has.
+The main UI to use Watermelon is the context box you see when you hover over lines of code with our extension installed. This UI is complemented with the sidebar we have.
+
+In the image below you can see how hovering over a line of code provides us the following data:
+
+![hover](https://i.ibb.co/rdvgVnS/hover-showcase.gif)
+
+- The most relevant PR for that line
```
Comment (EstebanDalelR) [inline README.md:30]: Is it there yet? 
That hasn't landed AFAIK

File: README.md [hunk 3]
```diff
@@ -11,16 +11,27 @@
...
+- The latest commit message for that line
+- The level of churn for the file that you're currently in
```
Comment (EstebanDalelR) [inline README.md:32]: I'd say "churn" is not a layman word, even for experienced devs

File: README.md [hunk 4]
```diff
@@ -11,16 +11,27 @@
...
+
+If you want to do a deeper reserach about the code context history, simply click one of the buttons in the context box.
```
Comment (EstebanDalelR) [inline README.md:34]:
```suggestion
If you want to do a deeper research about the code context history, simply click one of the links in the context box.
```

File: README.md (hunk 3 above)
Comment (EstebanDalelR) [inline README.md:32]:
```suggestion
- The amount of changes for the file that you're currently in, usually called churn
```

File: README.md (hunk 2 above)
Comment (EstebanDalelR) [inline README.md:30]:
```suggestion

```

File: README.md [hunk 5]
```diff
@@ -11,16 +11,27 @@
...
 - View Git Blame
 - View Pull Requests
+- Daily Summary
 
-Simply click the Watermelon icon on the sidebar. After that you can click any of the 2 different buttons Watermelon has.
+The main UI to use Watermelon is the context box you see when you hover over lines of code with our extension installed. This UI is complemented with the sidebar we have.
+
+In the image below you can see how hovering over a line of code provides us the following data:
+
+![hover](https://i.ibb.co/rdvgVnS/hover-showcase.gif)
```
Comment (EstebanDalelR) [inline README.md:28]: Shouldn't these media files live in [`/images`](https://github.com/watermelontools/wm-extension/tree/dev/media)?

File: README.md [hunk 6]
```diff
@@ -44,6 +55,14 @@ Alternatively, you can <a href="https://github.com/watermelontools/wm-extension#
 
 You may also highlight and right click on the code, you will find the 'Get Pull Requests with Watermelon' command at the end.
 
+### Daily Summary
+
+Get an overview of which issues and PR reviews are assigned to you. Both on the current repository and across all GitHub. 
+
+To use it, open the Watermelon sidebar.
+
+![daily-summary](https://i.ibb.co/RCym1hF/Screen-Shot-2022-07-13-at-5-03-48-PM.png)
```
Comment (EstebanDalelR) [inline README.md:64]: Shouldn't these media files live in [/images](https://github.com/watermelontools/wm-extension/tree/dev/media)? Or our webpage?
If we have it at our web page we may edit it without a PR

**Review by EstebanDalelR — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (baristaGeek) [reply README.md:30]: Aren't we rendering a PR on the box?

File: README.md (hunk 1 above)
Comment (baristaGeek) [reply README.md:22]: This will not be merged until #350 is

File: README.md (hunk 3 above)
Comment (baristaGeek) [reply README.md:32]: propose a change please

File: README.md (hunk 2 above)
Comment (EstebanDalelR) [reply README.md:30]: I looked at the image that was linked and I see the text "The latest commit is [...] by {user} on {date}", am I missing something?

File: README.md (hunk 3 above)
Comment (EstebanDalelR) [reply README.md:32]: https://github.com/watermelontools/wm-extension/pull/352/commits/e9231a437999c8995aaee0e79f0755502b60c7ad#r920584199

File: README.md (hunk 2 above)
Comment (EstebanDalelR) [reply README.md:30]: Ah, I see another hover, but it's only for the first 2 frames, hard to catch.
Screen Shot 2022-07-13 at 6 32 43 PM

File: README.md (hunk 6 above)
Comment (baristaGeek) [reply README.md:64]: I'm on the Webflow dashboard and don't see a straightforward way to do it. Will place it on /images

Comment (EstebanDalelR) [reply README.md:64]: Thank you.

File: README.md (hunk 2 above)
Comment (EstebanDalelR) [reply README.md:30]: That is a Squash commit.

Comment (EstebanDalelR) [reply README.md:30]: If you run it, you will get a huge comment title with a lot of issue numbers (#352) for example.
That is a github specific behaviour

Comment (baristaGeek) [reply README.md:30]: Gotcha

**Review by EstebanDalelR — APPROVED** [review]
I would remove the hover PR thing, other than that, seems ok.

**Pull request merged**